Serialization.md

```diff
@@ -0,0 +1,184 @@
+
+```
+ <FormulaCache>
+          <Formulas>
+            <Formula id="nyGiXWaSW0OYhQJ__lyZhg" name="Concentration_formula" dim="Concentration (molar)" formula="M/V">
+              <Paths>
+                <Path path="0" as="1" dim="2" />
+                <Path path="3" as="4" dim="5" />
+              </Paths>
+            </Formula>
+            <Formula id="Jv7-T-BLjEKUKypb2n2YIA" name="PLnL_zu_MFormula" dim="Concentration (molar)" formula="1e15/6.02214179e23" />
+            <Formula id="lU8mEpGab0-unqhumPvr6g" name="PLFormula" dim="Concentration (molar)" formula="dilutionFactor*S_PL*PLnL_zu_M">
+              <Paths>
+                <Path path="6" as="7" />
+                <Path path="8" as="9" />
+                <Path path="10" as="11" />
+              </Paths>
+            </Formula>
+            .
+            .
+            .
+         </Formulas>
+```
+The above excerpt is a simple PK-Sim project, and more specifically the [S1_concentrBased.pkml](https://github.com/Open-Systems-Pharmacology/PK-Sim/blob/develop/tests/PKSim.Tests/Data/S1_concentrBased.pkml) that is part of the test data of PKSim codebase.
+As you can see, in the Formula Cache, instead of having the actual formula strings, we have path numbers that refer to the StringMap that follows:
+
+```
+ <StringMap>
+            <Map s=".." id="0" />
+            <Map s="M" id="1" />
+            <Map s="Amount" id="2" />
+            <Map s="..|..|Volume" id="3" />
+            <Map s="V" id="4" />
+            <Map s="Volume" id="5" />
+            <Map s="..|PL|dilutionFactor" id="6" />
+            <Map s="dilutionFactor" id="7" />
+            <Map s="..|PL|S_PL" id="8" />
+            <Map s="S_PL" id="9" />
+            .
+            .
+            .
+ </StringMap>
+```
+
+This has occurred historically in order to avoid duplication of strings in bigger project files and thus help reduce the project file size. 
+
+## Xml Attribute Mapping
+
+The functionality of mapping to specific xml attributes is handled by AttributeMappers that are added to the AttributeMapperRepository either in [OSPSuiteXmlSerializerRepository](https://github.com/Open-Systems-Pharmacology/OSPSuite.Core/blob/develop/src/OSPSuite.Core/Serialization/Xml/OSPSuiteXmlSerializerRepository.cs) in OSPSuite.Core or in the individual solutions in [PKSimXmlSerializerRepository](https://github.com/Open-Systems-Pharmacology/PK-Sim/blob/develop/src/PKSim.Infrastructure/Serialization/Xml/Serializers/PKSimXmlSerializerRepository.cs) and [MoBiXmlSerializerRepository](https://github.com/Open-Systems-Pharmacology/MoBi/blob/develop/src/MoBi.Presentation/Serialization/Xml/Serializer/MoBiXmlSerializerRepository.cs). This is how for example you can specify for the `Species` in PK-Sim the name gets serialized in the xml and the rest gets saved in the database.
```

Comment:
I understand that the MoBi file path could change with OSMOSES- but generally the file paths for serilaization of projects would remain pretty constant I guess and I find the link to github helpful. btw @msevestre why is the MoBiXmlSerializerRepository.cs in Presentation?